schema/identifier.json

```diff
@@ -8,7 +8,7 @@
       "ISO 19115-2": ["MD_Identifier"]
     },
     "type": "object",
-    "required": ["identifier"],
+    "required": ["identifier", "namespace"],
```

Comment:
@dwalt was "namespace" supposed to be optional?

Reply:
@hmaier-fws Yes, per user request. A warning status would not require entry. Are you wanting to leave it completely optional, without a warning?